cves/CVE-2012-4520.yml

```diff
@@ -1,300 +1,323 @@
+description_instructions: |
+    You can get an initial description from the CVE entry on cve.mitre.org. These
+    descriptions are a fine start, but they can be kind of jargony.
+    
+    Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
+    read to anyone with some programming experience. We can always pull up the NVD
+    description later to get more technical.
+    
+    Try to still be specific in your description, but remove project-specific
+    stuff. Remove references to versions, specific filenames, and other jargon
+    that outsiders to this project would not understand. Technology like "regular
+    expressions" is fine, and security phrases like "invalid write" are fine to
+    keep too.
+    
+    Your target audience is people just like you before you took any course in
+    security
+discovered: 
+  answer: "The vulnerability was discovered by James Kettle. There is no explanation for how it was found. https://www.djangoproject.com/weblog/2012/oct/17/security/\n"
+  automated: false
+  contest: false
+  developer: true
   question: |
-    Were automated unit tests involved in this vulnerability?
-    Was the original code unit tested, or not unit tested? Did the fix involve
-    improving the automated tests?
-
-    For code: and fix: - your answer should be boolean.
-
-    For the code_answer below, look not only at the fix but the surrounding
-    code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
-
-    For the fix_answer below, check if the fix for the vulnerability involves
-    adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
-discovered:
+      How was this vulnerability discovered?
+      
+      Go to the bug report and read the conversation to find out how this was
+      originally found. Answer in longform below in "answer", fill in the date in
+      YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+      employee (you can tell from their email address). If it's clear that the
+      vulenrability was discovered by a contest, fill in the name there.
+      
+      The automated, contest, and developer flags can be true, false, or nil.
+      
+      If there is no evidence as to how this vulnerability was found, then please explain where you looked.
+fixes: 
+  - 
+    commit: b45c377f8f488955e0c7069cad3f3dd21910b071
+    note: ~
+  - 
+    commit: 92d3430f12171f16f566c9050c40feefb830a4a3
+    note: ~
+fixes_vcc_instructions: |
+    Please put the commit hash in "commit" below (see my example in
+    CVE-2011-3092.yml). Fixes and VCCs follow the same format.
+i18n: 
+  answer: false
+  note: "the vulnerability impacts displayed URL which has no variation depending on user location.\n"
```

Comment:
Could you remove the manual line break characters you have in these strings and un-quote them?  Both are unnecessary.  If you want to span multiple lines, you can use a pipe character `|` after the colon like shown in the other blocks.